src/compiler/compiler.lisp

```diff
@@ -1310,6 +1310,12 @@
 (define-builtin /log (x)
   `(method-call |console| "log" ,x))
 
+(define-raw-builtin %console (x &rest y)
```

Comment:
We have a couple of similar built'ins

```lisp
(define-builtin /debug (x)
  `(method-call |console| "log" (call-internal |xstring| ,x)))

(define-builtin /log (x)
  `(method-call |console| "log" ,x))
```

would some of them work instead of adding a new one?

Reply:
This is a debug function that I put in my dev every time.

`(% console "msg # ~ d. obzh ~ o ~ d" msg-id o1 o2)`

https://developer.mozilla.org/en-US/docs/Web/API/Console  see `Outputting multiple objects`

Reply:
I see. So if the issue is just the multiple arguments, could we extend `/log` instead then?  Or replace it with other one. But it is not going to be very clear which one is best to use otherwise.

Reply:
`eval-always` this is a my lazy version `(eval-when (:compile-toplevel :load-toplevel :execute)`

Does it make sense to compile the entire `CLOS` twice, first on the `host` and then into the `target`? The first version was done just like that by me. I hope no one will rewrite the compiler on `generics` forms ? Then it would make sense `:both` mode.

Only those functions are highlighted that are sufficient and necessary to perform `(boot-stage-N)` functions to revive the `CLOS` and use it while loading the bandl. I'm not sure that such subtleties need to be commented on. This will have to be done in many files 😊